GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -160,7 +186,15 @@ void Renderer::renderCamera(Scene* scene, Camera* camera, int framebufferId,
         }
 
         glBindFramebuffer(GL_FRAMEBUFFER, framebufferId);
-        glViewport(viewportX, viewportY, viewportWidth, viewportHeight);
+
+        // ---------------------------------------------AR------------------------{
+        if (setIsAr != 0)
+            glViewport(viewportX + ar_offsets[0], viewportY + ar_offsets[1],
```

Comment:
Please add bracket here and for the else clause even though it is one line call.